app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java

```diff
@@ -59,14 +60,14 @@ protected void onCreate(Bundle savedInstanceState) {
                 (CollapsingToolbarLayout) findViewById(R.id.collapsing_toolbar);
         collapsingToolbar.setTitle(track);
         sessionsRecyclerView = (RecyclerView) findViewById(R.id.recyclerView);
-        sessionsListAdapter = new SessionsListAdapter(dbSingleton.getSessionbyTracksname(track));
-
+        trackSessionsListAdapter = new TrackSessionsListAdpater(dbSingleton.getSessionbyTracksname(track));
+        trackSessionsListAdapter.setTrackId(dbSingleton.getTrackbyName(track).getId());
```

Comment:
if the constructor is used to initialise the adapter then these 2 statements can be reduced into 1.